lib/measured/conversion_builder.rb

```diff
@@ -0,0 +1,35 @@
+class Measured::ConversionBuilder
+  def initialize
+    @base_unit = nil
+    @units = []
+    @case_sensitive = false
+  end
+
+  def base(unit_name, aliases: [])
+    raise Measured::UnitError, "Setting #{unit_name} as base unit, but already defined as #{@base_unit}." if @base_unit
+    @base_unit = Measured::Unit.new(unit_name, aliases: aliases)
+  end
+
+  def unit(unit_name, aliases: [], value:)
+    @units << Measured::Unit.new(unit_name, aliases: aliases, value: value)
+  end
+
+  def case_sensitive(is_case_sensitive)
+    @case_sensitive = is_case_sensitive
```

Comment:
Always avoid `is_` in ruby. Set this to be `value` or something. And if it should always be a boolean, coerce it to be so:

```ruby
def case_sensitive(value)
  @case_sensitive = !!value
end
```